src/middleware/validation.ts

```diff
@@ -0,0 +1,49 @@
+import { body, validationResult, ValidationChain } from "express-validator";
+
+export const validateParticipantFields: ValidationChain[] = [
+  body("participantParams.name")
```

Comment:
Is there a reason why this is called 'participantParams' and not just 'participant'?

Reply:
Well, no,  the JSON that I send is in that format : 
{
  "participantParams": {
    "name": "Martina",
    "phone_number": "123456789",
    "email": "test@example.com"
  },
  "companyId": 2
}

Reply:
Agreed, will change that